spec/controllers/coronavirus_form/get_help_from_nhs_controller_spec.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+RSpec.describe CoronavirusForm::GetHelpFromNhsController, type: :controller do
```

Comment:
It could be worth changing this to a request test to bring it in line with the rest of the views? There are some good examples in https://github.com/alphagov/govuk-coronavirus-find-support/tree/master/spec/requests

Reply:
Moved to request tests. We still have the accessibility statement, privacy notice and session expired as controller tests, but I guess that's not a priority now.

Reply:
Yep, good spot - I'll move them over now. Thanks!